src/test/java/net/minecraftforge/debug/world/BiomeLoadingEventTest.java

```diff
@@ -38,6 +40,6 @@ public BiomeLoadingEventTest(){
     }
     public void onBiomeLoading(BiomeLoadingEvent event){
         ResourceLocation biome = event.getName();
-        LOGGER.info(String.format("Biome loaded: %s", biome.toString()));
+        LOGGER.info(String.format(Locale.ENGLISH, "Biome loaded: %s", biome.toString()));
```

Comment:
This use of `String.format` should be removed in favor of using logger parameter substitution:
```suggestion
        LOGGER.info("Biome loaded: %s", biome.toString());
```